src/main/java/de/srendi/advancedperipherals/common/util/ChunkManager.java

```diff
@@ -204,19 +276,26 @@ public static LoadChunkRecord deserialize(@NotNull CompoundTag tag) {
             return dimensionName;
         }
 
+        public int getRadius() {
+            return radius;
+        }
+
+        public void setRadius(int radius) {
+            this.radius = radius;
+        }
+
         public void touch() {
-            lastTouch = LocalDateTime.now().toEpochSecond(ZoneOffset.UTC);
+            lastTouch = tickCounter;
         }
 
         public boolean isValid() {
-            long currentEpoch = LocalDateTime.now().toEpochSecond(ZoneOffset.UTC);
-            return lastTouch + APConfig.PERIPHERALS_CONFIG.chunkLoadValidTime.get() >= currentEpoch;
+            return lastTouch + APConfig.PERIPHERALS_CONFIG.chunkLoadValidTime.get() * 20 >= tickCounter;
         }
-
```

Comment:
Can we keep the empty line just for better looking

Reply:
Yeah ofc
Not sure why that got removed

Reply:
wait I see there is `modBus.addListener(ChunkManager::registerTicketController)`